std/php/Global.hx

```diff
@@ -931,6 +931,21 @@ extern class Global {
 	**/
 	static function header(string:String, replace:Bool = true, ?http_response_code:Int):Void;
 
+	/**
+		@see http://php.net/manual/en/function.header-remove.php
+	**/
+	static function header_remove(?name:String):Void;
+
+	/**
+		@see http://php.net/manual/en/function.headers-list.php
+	**/
+	static function headers_list():Array<String>;
```

Comment:
Return type should be `NativeIndexedArray<String>`

Reply:
Of course! :sweat_smile: Fixed.